build.gradle

```diff
@@ -3,7 +3,7 @@ plugins {
     id 'org.jetbrains.kotlin.jvm' version '1.4.31'
     id 'checkstyle'
     id 'com.github.alisiikh.scalastyle' version '3.3.1'
-    id 'org.jetbrains.intellij' version '0.4.21'
+    id 'org.jetbrains.intellij' version '0.7.2'
```

Comment:
I can't remember what it does, but it can't hurt to have a newer version right?

Reply:
I can comment here.

> What does this do?

This is about IntelliJ's plugin for Gradle (building tool). It adds tasks such as `runIde` and `buildPlugin` to Gradle. It should be (more or less) independent of which version of IntelliJ we target. Why is the version ungraded now, I can't tell, but

> it can't hurt to have a newer version right?

If it builds, it should be fine. (In general, it _can_ hurt to have a newer version.)